web/public/js/services/event-utils.js

```diff
@@ -51,8 +51,10 @@ angular.module('cpZenPlatform').factory('eventUtils', ['$translate', function($t
   eventUtils.getFormattedDates = function (event) {
     var utcOffset = moment().utcOffset();
 
-    var startDate = moment.utc(_.head(event.dates).startTime).subtract(utcOffset, 'minutes').toDate();
-    var endDate = moment.utc(_.head(event.dates).endTime).subtract(utcOffset, 'minutes').toDate();
+    var mStartDate = moment.utc(_.head(event.dates).startTime);
+    var mEndDate = moment.utc(_.head(event.dates).endTime);
+    var startDate = mStartDate.subtract(mStartDate.utcOffset(), 'minutes').toDate();
```

Comment:
We're now getting the UTC offset from the date we're displaying and subtracting that instead of the UTC offset for the current date.